docs/lab/gce/debian-driver.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: kmm.sigs.k8s.io/v1beta1
+kind: Module
+metadata:
+  name: kmm-ci-deb-a
+spec:
+  moduleLoader:
+    container:
+      modprobe:
+        moduleName: kmm-ci-deb-a 
+
+      kernelMappings:
+        - literal: 5.10.127 
+          containerImage: quay.io/ebelarte/kmmo:gke-kmm-deb-1 
```

Comment:
We should probably use a generic image here like `quay.io/org/image:gke-kmm-deb-1` or something similar.